### PR TITLE
Fix for #835

### PR DESCRIPTION
**Pull request by exp0logy**
Added Exception for email not found and Modified HTTP error code to unauth (401) rather than forbidden (403).

Testing works on my live copy.

### CODE REVIEW

**Review by BelleNottelling — APPROVED** [review]
Seems reasonable

File: src/modules/Client/Api/Guest.php [hunk 1]
```diff
@@ -138,10 +138,13 @@ public function login($data)
 
         $service = $this->getService();
         $client = $service->authorizeClient($data['email'], $data['password']);
+        
+        if (!$service->clientAlreadyExists($data['email'])) {
+            throw new \Box_Exception('Email not found in client database.', [], 401);
```
Comment (jaapmarcus) [inline src/modules/Client/Api/Guest.php:143]: Errors should be the same as line 147

Just to prevent users to "discover" email addresses by quessing

**Review by jaapmarcus — CHANGES_REQUESTED** [review]
See comment

**Comment by exp0logy** [comment]
Changes made. Removed first exception block as the bottom one will have the same effect :)

**Review by jaapmarcus — APPROVED** [review]

**Pull request merged**